### PR TITLE
Github: updated Contributing.md

### PR DESCRIPTION
**Pull request by innocuo**
- Added some guidelines based on suggested ideas from [the forum](https://discourse.joplinapp.org/t/contributing-md-suggestions/2317/8)
- Moved some rules to bulleted lists (for quicker reading).
- When possible, moved the main idea of sentences to the beginning of the sentences (for example: "The _Joplin Forum_ is the...place for user support", instead of "For general support, software questions...go to the _Joplin Forum_")

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,26 +1,42 @@
 # User support
 
-For general discussion about Joplin, user support, software development questions, and to discuss new features, please go to the [Joplin Forum](https://discourse.joplin.cozic.net/). It is possible to login with your GitHub account.
+The [Joplin Forum](https://discourse.joplin.cozic.net/) is the community driven place for user support, general discussion about Joplin, problems with installation, new features and software development questions. It is possible to login with your GitHub account.
```
Comment (tessus) [inline CONTRIBUTING.md]: We should use the new link: https://discourse.joplinapp.org/

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,26 +1,42 @@
...
+- Stay on topic, but describe the issue in detail so that others can reproduce it.
+- **Provide a screenshot** if possible. A screenshot showing the problem is often more useful than a paragraph describing it.
 
 # Feature requests
 
-Again, please check that it has not already been requested. If it has, simply **up-vote the issue** - the ones with the most up-votes are likely to be implemented. "+1" comments are not tracked.
+Please check that your request has not already been posted in the [Github Issue Tracker](https://github.com/laurent22/joplin/issues?utf8=%E2%9C%93&q=is%3Aissue). If it has, **up-voting the issue** increases the chances it'll be noticed and implemented in the future. "+1" comments are not tracked.
 
-# Creating a pull request
+As a general rule, suggestions to _improve Joplin_ should be posted first in the [Joplin Forum](https://discourse.joplin.cozic.net/) for discussion.
```
Comment (tessus) [inline CONTRIBUTING.md]: new Joplin Forum address: https://discourse.joplinapp.org/

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,26 +1,42 @@
...
+## Contributing to Joplin's translation
+
+Joplin is available in multiple languages thanks to the help of its users. You can help translate Joplin to your language or keep it up to date. Please read the documentation about [Localisation](https://github.com/laurent22/joplin#localisation).
+
+## Contributing to Joplin's code
+
+If you want to start contributing to the project's code, please follow these guidelines before creating a pull request: 
+
+- Bug fixes are always welcome. Start by reviewing the list of [essential issues](https://github.com/laurent22/joplin/issues?q=is%3Aissue+is%3Aopen+label%3Aessential)
+- Before adding a new feature, ask about it in the [Github Issue Tracker](https://github.com/laurent22/joplin/issues?utf8=%E2%9C%93&q=is%3Aissue) or the [Joplin Forum](https://discourse.joplin.cozic.net/), or check if existing discussions exist to make sure the new functionality is desired.
```
Comment (tessus) [inline CONTRIBUTING.md]: new Joplin Forum address: https://discourse.joplinapp.org/

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,26 +1,42 @@
...
+- **Changes that will consist in more than 50 lines of code should be discussed the [Joplin Forum](https://discourse.joplin.cozic.net/)**, so that you don't spend too much time implementing something that might not be accepted.
```
Comment (tessus) [inline CONTRIBUTING.md]: Changes that consist of more than 50 lines of code should be discussed in the ...

new Joplin Forum address: https://discourse.joplinapp.org/

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,26 +1,42 @@
...
 
 # Reporting a bug
 
-Please check first that it [has not already been reported](https://github.com/laurent22/joplin/issues?utf8=%E2%9C%93&q=is%3Aissue). Also consider [enabling debug mode](https://github.com/laurent22/joplin/blob/master/readme/debugging.md) before reporting the issue so that you can provide as much details as possible to help fix it.
+File bugs in the [Github Issue Tracker](https://github.com/laurent22/joplin/issues?utf8=%E2%9C%93&q=is%3Aissue). Please follow these guidelines:
 
-If possible, **please provide a screenshot**. A screenshot showing the problem is often more useful than a paragraph describing it as it can make it immediately clear what the issue is.
+- Search existing issues first, make sure yours hasn't already been reported.
+- Don't use the issue tracker for support questions.
+- Consider [enabling debug mode](https://github.com/laurent22/joplin/blob/master/readme/debugging.md) before so that you can provide as much details as possible when reporting the issue.
```
Comment (tessus) [inline CONTRIBUTING.md]: we can remove the word `before`

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,26 +1,42 @@
...
 
-- If you want to add a new feature, consider asking about it before implementing it or checking existing discussions to make sure it is within the scope of the project. As a rule of thumb **if your change is likely to involve more than 50 lines of code, you should discuss it in the forum**, just so that you don't spend too much time implementing something that might not be accepted.
+Avoid listing multiple requests in one report in the [Github Issue Tracker](https://github.com/laurent22/joplin/issues?utf8=%E2%9C%93&q=is%3Aissue). One issue per request makes it easier to track and discuss it.
 
-- Bug fixes are always welcome.
+# Contribute to the project
+
+## Contributing to Joplin's translation
+
+Joplin is available in multiple languages thanks to the help of its users. You can help translate Joplin to your language or keep it up to date. Please read the documentation about [Localisation](https://github.com/laurent22/joplin#localisation).
```
Comment (tessus) [inline CONTRIBUTING.md]: Not sure, if we should use the repo URL or the web site: https://joplinapp.org/#localisation

@laurent22 what do you think?

**Comment by innocuo** [comment]
I updated the forum links and removed the word "before" as requested.

File: CONTRIBUTING.md (hunk 6 above)
Comment (tessus) [reply CONTRIBUTING.md]: Ok, we can leave this for now. We can always change this later, if necessary.

**Comment by tessus** [comment]
Thanks for the changes

**Comment by laurent22** [comment]
Looks good, thanks @innocuo!

**Pull request merged**